containers/check-release/src/check-release.sh

```diff
@@ -96,9 +96,9 @@ printf "\n==== Dist GPG Signatures ====\n"
 find $DIST_DIR -type f ! -name '*.sha512' ! -name '*.asc' \
 	-exec bash -c "gpg --verify '{}.asc' '{}' $PRINT_FIND_RESULT" \;
 
-printf "\n==== RPM Signatures ====\n"
+printf "\n==== RPM Embedded Signatures ====\n"
 find $DIST_DIR -type f -name '*.rpm' \
-	-exec bash -c "rpm -K '{}' $PRINT_FIND_RESULT" \;
+	-exec bash -c "rpm -K '{}' | grep 'signatures OK' $PRINT_FIND_RESULT" \;
```

Comment:
Per "... we grep for the string "signatures OK"--this is only output if both conditions hold."

The place that outputs this "signatures OK" (not modified in this change set) needs doc/comments to indicate that it is output only if both conditions hold since we're depending on that invariant here... unless it already has them.

Reply:
This is checking the output of `rpm -K`. I've added a detailed comment explaining what rpm -K does because isn't clear at all, and it wouldn't surprise me if rpm changes this behavior some day--it's not really documented by rpm anywhere that I've found.

I've also modified it to grep for "digests signatures OK" to also require that digests exist. In practice, these digest always exists since rpmbuild always creates them. It's really only the signatures that are optional, but it's good to verify and a small change.